source/dub/project.d

```diff
@@ -1242,6 +1246,9 @@ void processVars(ref BuildSettings dst, in Project project, in Package pack,
 	dst.addPreRunCommands(settings.preRunCommands);
 	dst.addPostRunCommands(settings.postRunCommands);
 
+	if (!settings.injectSourceFiles.empty)
```

Comment:
why the check if there is any element? processVars should not do any work if it's empty and the other parts here don't use it either.

Reply:
Okay, that can go.